sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -886,7 +889,13 @@ class Analyzer(
         }
 
       case u: UnresolvedRelation =>
-        lookupRelation(u.multipartIdentifier).map(resolveViews).getOrElse(u)
+        lookupRelation(u.multipartIdentifier)
+          .map {
+            case r: DataSourceV2Relation => SubqueryAlias(u.multipartIdentifier, r)
```

Comment:
seems better to add `SubqueryAlias` in `lookupRelation`, to be consistent with v1 relation.

Reply:
sounds good. updated.